inc/class-astra-dynamic-css.php

```diff
@@ -3569,7 +3572,10 @@ public static function related_posts_static_css() {
 				width: 100%;
 				position: relative;
 			}
-			.ast-related-post-featured-section + .entry-header, .ast-related-post-featured-section + .entry-content {
+			.ast-related-posts-wrapper .ast-related-posts-inner-section {
```

Comment:
```suggestion
			.ast-related-posts-inner-section {
```
@imnavanath I think we can reduce the CSS weightage here

Reply:
Done 👍